### PR TITLE
Added meeting calendar under resources

### PR DESCRIPTION
**Pull request by noatamir**
👋  
1. I added the meeting calendar under resources, highlighting the new contributor meeting. 
2. I wasn't able to preview the CSS locally so please let me know if I added the `fa-icon` correctly or not.
3. I also tweaked the text for Gitter to be more inline with what we have in the README on github. Let me know if you'd like me to revert that.
4. I put those to together. I thought the order made sense because the first 3 are for all project users, and the last 2 are relevant for those interested specifically in contributing to the project.

### CODE REVIEW

**Comment by QuLogic** [comment]
It looks like the icon is not working?
image

It's FontAwesome v5; I think you might have used a v6 icon name. It probably needs the `callout__icon` style applied as well.

File: docs/body.html [hunk 1]
```diff
@@ -230,6 +221,24 @@ <h3>Resources</h3>
               >.
             </p>
           </div>
+
+          <div class="callout__list">
+            <i class="fab fa-solid fa-calendar-days"aria-hidden="true"></i>
```
Comment (QuLogic) [inline docs/body.html:226]: I think this the equivalent v5 icon:
```suggestion
            <i class="fa fa-solid fa-calendar-alt" aria-hidden="true"></i>
```

Comment (noatamir) [reply docs/body.html:226]: ah thanks. I just saw the other icons were added as svgs and was working on adding on adding this one as well. Is that preferred?

Comment (QuLogic) [reply docs/body.html:226]: I think those must be old; I don't see them referenced anywhere.

**Comment by QuLogic** [comment]
Oops, I forgot to put `callout__icon` in my suggested change.

**Comment by noatamir** [comment]
I can fix it 😉

File: docs/body.html [hunk 2]
```diff
@@ -230,6 +221,24 @@ <h3>Resources</h3>
               >.
             </p>
           </div>
+
+          <div class="callout__list">
+            <i class="fa fa-solid fa-calendar-alt" aria-hidden="true"></i>
```
Comment (noatamir) [inline docs/body.html:226]:
```suggestion
            <i class="fa fa-solid fa-calendar-alt callout__icon" aria-hidden="true"></i>
```

**Comment by noatamir** [comment]
LGTM now. Thanks for the assist 😃

**Comment by noatamir** [comment]
(some day we should make all these links open in new tabs right?!)

**Comment by jklymak** [comment]
> (some day we should make all these links open in new tabs right?!)

I would be pretty strenuously against that.  I don't think we should force users to open tabs, and all browsers have a keyboard shortcuts to open in new tab if the user wishes.  The only time it makes sense to open a new tab is if navigating a way will destroy the state of a page, which is definitely not the case for our home page.

**Comment by jklymak** [comment]
BTW I squash merged this....

**Pull request merged**